File: GTTS/INL.SearchService/Models/SearchParticipants_Item.cs
using System;

namespace INL.SearchService.Models
{
    public class SearchParticipants_Item : ISearchParticipants_Item
    {
        public string ParticipantType { get; set; }
        public long PersonID { get; set; }
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        public DateTime? DOB { get; set; }
        public char? Gender { get; set; }
        public string JobTitle { get; set; }
        public string JobRank { get; set; }
        public int? CountryID { get; set; }
        public string CountryName { get; set; }
        public string CountryFullName { get; set; }
        public long? UnitID { get; set; }
        public string UnitName { get; set; }
        public string UnitNameEnglish { get; set; }
        public long? UnitMainAgencyID { get; set; }
        public string AgencyName { get; set; }
        public string AgencyNameEnglish { get; set; }
        public string VettingStatus { get; set; }
        public DateTime? VettingStatusDate { get; set; }
        public string VettingType { get; set; }
        public string Distinction { get; set; }
        public DateTime? EventStartDate { get; set; }
        public long TrainingEventID { get; set; }
        public long TrainingEventParticipantID { get; set; }
        public bool? IsParticipant { get; set; }
        public bool RemovedFromEvent { get; set; }
        public string DepartureCity { get; set; }
        public DateTime? DepartureDate { get; set; }
        public DateTime? ReturnDate { get; set; }
        public long? PersonsVettingID { get; set; }
        public bool IsTraveling { get; set; }
        public bool? OnboardingComplete { get; set; }
        public int? VisaStatusID { get; set; }
        public string VisaStatus { get; set; }
        public string ContactEmail { get; set; }
        public bool? IsUSCitizen { get; set; }
        public long? TrainingEventGroupID { get; set; }
        public string GroupName { get; set; }
        public int? VettingPersonStatusID { get; set; }
        public string VettingPersonStatus { get; set; }
        public DateTime? VettingPersonStatusDate { get; set; }
        public byte? VettingBatchTypeID { get; set; }
        public string VettingBatchType { get; set; }
        public int? VettingBatchStatusID { get; set; }
        public string VettingBatchStatus { get; set; }
        public DateTime? VettingBatchStatusDate { get; set; }
        public bool? RemovedFromVetting { get; set; }
        public string VettingTrainingEventName { get; set; }
        public bool? IsLeahyVettingReq { get; set; }
        public bool? IsVettingReq { get; set; }
        public bool? IsValidated { get; set; }
        public long? PersonsUnitLibraryInfoID { get; set; }
        public string NationalID { get; set; }
        public int? UnitTypeID { get; set; }
        public string UnitAcronym { get; set; }
        public string UnitParentName { get; set; }
        public string UnitParentNameEnglish { get; set; }
        public string UnitType { get; set; }
        public int? DocumentCount { get; set; }
        public string CourtesyVettingsJSON { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/search-participants_item.ts


import { ISearchParticipants_Item } from './isearch-participants_item';

export class SearchParticipants_Item implements ISearchParticipants_Item {
  
	public ParticipantType: string = "";
	public PersonID: number = 0;
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public DOB?: Date;
	public Gender?: string;
	public JobTitle: string = "";
	public JobRank: string = "";
	public CountryID?: number;
	public CountryName: string = "";
	public CountryFullName: string = "";
	public UnitID?: number;
	public UnitName: string = "";
	public UnitNameEnglish: string = "";
	public UnitMainAgencyID?: number;
	public AgencyName: string = "";
	public AgencyNameEnglish: string = "";
	public VettingStatus: string = "";
	public VettingStatusDate?: Date;
	public VettingType: string = "";
	public Distinction: string = "";
	public EventStartDate?: Date;
	public TrainingEventID: number = 0;
	public TrainingEventParticipantID: number = 0;
	public IsParticipant?: boolean;
	public RemovedFromEvent: boolean = false;
	public DepartureCity: string = "";
	public DepartureDate?: Date;
	public ReturnDate?: Date;
	public PersonsVettingID?: number;
	public IsTraveling: boolean = false;
	public OnboardingComplete?: boolean;
	public VisaStatusID?: number;
	public VisaStatus: string = "";
	public ContactEmail: string = "";
	public IsUSCitizen?: boolean;
	public TrainingEventGroupID?: number;
	public GroupName: string = "";
	public VettingPersonStatusID?: number;
	public VettingPersonStatus: string = "";
	public VettingPersonStatusDate?: Date;
	public VettingBatchTypeID?: number;
	public VettingBatchType: string = "";
	public VettingBatchStatusID?: number;
	public VettingBatchStatus: string = "";
	public VettingBatchStatusDate?: Date;
	public RemovedFromVetting?: boolean;
	public VettingTrainingEventName: string = "";
	public IsLeahyVettingReq?: boolean;
	public IsVettingReq?: boolean;
	public IsValidated?: boolean;
	public PersonsUnitLibraryInfoID?: number;
	public NationalID: string = "";
	public UnitTypeID?: number;
	public UnitAcronym: string = "";
	public UnitParentName: string = "";
	public UnitParentNameEnglish: string = "";
	public UnitType: string = "";
	public DocumentCount?: number;
	public CourtesyVettingsJSON: string = "";
  
}


